core/tests/release_sources/feconf.txt

```diff
@@ -19,3 +19,8 @@ CURRENT_COLLECTION_SCHEMA_VERSION = 4
 
 # The current version of story contents dict in the story schema.
 CURRENT_STORY_CONTENTS_SCHEMA_VERSION = 1
+
+INCOMING_EMAILS_DOMAIN_NAME = ''
+ADMIN_EMAIL_ADDRESS = 'system@example.com'
+SYSTEM_EMAIL_ADDRESS = 'system@example.com'
```

Comment:
In these test files, use different email addresses for admin + system. Otherwise, if the code uses one constant when it should really use the other, that error won't be caught.

Reply:
Done.